standards/rmrk2.0.0/entities/base.md

```diff
@@ -0,0 +1,200 @@
+# Base
+
+A base is a meta-entity, stored on-chain or off chain. It is not an NFT, but can be thought of as
+the interface or abstract class for an NFT's render.
+
+A base is a JSON object with three properties: `id`, `type`, and `parts`. There is an implied
+`owner` field, matching the address that created the base.
+
+Example:
+
+```json=
+{
+    "id": "kanaria_superbird",
+    "type": "svg",
+    "parts": [

```

Comment:
not sure if this should be an array or just an object?

Reply:
How would just an object work when enumerating different parts?

Reply:
I just see in this example it's array with one object. and then this inner object has many objects. So would that array always have 1 item?

If we simply use object then wer can use Object.keys / Object.values, otherwise we can use array and move id as one of the fields

```
[{id: "bg", ...}, {id: "gem_1", ...}]
```

OR

```
{"bg": {...}, "gem_1": {...} }
```

Reply:
Oh good point, I missed that array leftover, but I think a full array with id as prop is actually better than keys. will change